src/components/Sum/Sum.jsx

```diff
@@ -1 +1,9 @@
-// export const Sum = () => ();
+export const Sum = ({ a = 0, b = 0 }) => {
+  const resultMessage = `Sum of ${a} and ${b} is ${a + b}`;
+
+  return (
+    <p>
+      {resultMessage}
+    </p>
```

Comment:
You could also write:
```suggestion
  <p>
    {`Sum of ${a} and ${b} is ${a + b}`}
  </p>
```